Update troubleshooting.md (#1288)
saucelabs/sauce-docs: docs/visual/e2e-testing/troubleshooting.md

```

Additional information on WebDriver timeouts can be found [here](https://developer.mozilla.org/en-US/docs/Web/WebDriver/Errors/ScriptTimeout).

> ## cropTo can not crop the target element

Reviewer: ```suggestion
## Can't Capture Snapshot with `cropTo`
```
Updates capitalization for headings.

Author: changed

> If you can't crop the target element with `cropTo` and the element isn't scrollable, you can use the `scrollAndStitchScreenshot` parameter. For example:

Reviewer: ```suggestion
When using the [Snapshot command](https://docs.saucelabs.com/visual/e2e-testing/commands-options/#snapshot-command), if you can't crop the target element with `cropTo` and the element isn't scrollable, you can use the `scrollAndStitchScreenshot` parameter. For example:
```
Adds a bit more context in the description of the issue.

Reviewer: We should also add the code snippet for the `scrollAndStitchScreenshot` option to this section: https://docs.saucelabs.com/visual/e2e-testing/commands-options/#snapshot-command

Author: Thank you for your rewording. Also added the new snippet in the snapshot-command page

```
await browser.execute('/*@visual.snapshot*/', 'Home', { scrollAndStitchScreenshot: true });
```

> ### limitation

Reviewer: ```suggestion
### Limitations
```

Author: Changed

> If the element is scrollable, we are unfortunately not supporting this feature for now.

Reviewer: ```suggestion
If the target element is scrollable, the `scrollAndStitchScreenshot` option isn't supported.
```

Author: when the element is scrollable, it's the cropTo option that won't work, I updated the description. Please review.

Reviewer: Thanks for clarifying. 

So if the target element isn't scrollable, we recommend to use `scrollAndStitchScreenshot`. And if the target element is scrollable, we also recommend to use `scrollAndStitchScreenshot` because `cropTo` may not work -- is that correct?

Author: If the target element isn't scrollable, we recommend to use scrollAndStitchScreenshot.

But if the target element is scrollable, this scrollAndStitchScreenshot and cropTo options won't work out.

Reviewer: Got it - thank you!

> If the target element is scrollable, the `cropTo` option will probably not capture all of its content.

Reviewer: ```suggestion
If the target element is scrollable, the `cropTo` option may not capture a complete screenshot.
```

Author: Thank you for your suggestion, updated.

```

Additional information on WebDriver timeouts can be found [here](https://developer.mozilla.org/en-US/docs/Web/WebDriver/Errors/ScriptTimeout).

> ### cropTo can not crop the target element

Author: Sorry about this, changed to use H2.

#### element is not scrollable

> In this case, you could try to use the scrollAndStitchScreenshot approach. For example:

Reviewer: If you're unable to crop the target element with `cropTo` and the element is not scrollable, you can use the `scrollAndStitchScreenshot` parameter. For example:

> In this case, you could try to use the scrollAndStitchScreenshot approach. For example:

Reviewer: ```suggestion
If you can't crop the target element with `cropTo` and the element isn't scrollable, you can use the `scrollAndStitchScreenshot` parameter. For example:```

Author: Thank you, changed the sentence.

```
await browser.execute('/*@visual.snapshot*/', 'Home', { scrollAndStitchScreenshot: true });
```

#### element is scrollable

> Unfortunately, we are not supporting this feature for now.

Reviewer: @xwa130saucer Are there any plans to support this in the near future?

Author: Sorry, we don't have plan to support this for now. I moved it to H3 heading as limitation. Does this look good?

Reviewer: Yes, looks good. Thanks!
